keps/sig-instrumentation/3077-contextual-logging/README.md

```diff
@@ -1037,6 +1079,11 @@ Revert commits that changed log calls.
 
 ## Implementation History
 
+* Kubernetes 1.24: initial alpha
+* Kubernetes 1.27: parts of kube-controller-manager converted
+* Kubernetes 1.28: kube-controller-manager converted completely, relationship
+  with log/slog in Go 1.21 clarified
+
```

Comment:
What about 1.30?

Reply:
We finished the kube-scheduler in 1.29. Let me add that.

For 1.30, we are focusing on client-go. We had a volunteer for kubelet and @tallclair wanted to do something with the apiserver, but it's unclear whether both will happen.

It shouldn't be in "Implementation History" anyway, should it? It's not history, it's the future...

Reply:
@soltysh: I added a "Status and next steps" section. I hope that also addresses your question about "estimates around how much work is left" - can you take another look?